paddle/fluid/operators/detail/grpc_server.cc

```diff
@@ -282,8 +282,9 @@ void AsyncGRPCServer::TryToRegisterNewSendOne() {
     VLOG(3) << "shutdown, do not TryToRegisterNewSendOne";
     return;
   }
-  RequestSend* send = new RequestSend(&service_, cq_send_.get(), sync_mode_,
-                                      scope_, &var_recv_queue_, dev_ctx_);
+  std::unique_ptr<RequestSend> send(
```

Comment:
@gongweibao can you take a look at this please, I remember gRPC can only delete the request object when it comes out from the `CompletionQueue`.

Reply:
Yep.You are right.